minsar/utils/download_asfserial_rsmas.py

```diff
@@ -57,8 +57,29 @@ def main(iargs=None):
         os.remove(os.path.expanduser('~') + '/.bulk_download_cookiejar.txt')
     except OSError:
         pass
-
-    generate_files_csv(project_slc_dir, inps.custom_template_file)
+    
+    dataset_template = Template(inps.custom_template_file)
+    dataset_template.options.update(PathFind.correct_for_ssara_date_format(dataset_template.options))
+    subprocess.Popen("rm new_files.csv", shell=True).wait()
+    standardTuple = (inps, dataset_template)
```

Comment:
Rename more descriptively please. Not sure what this is.

Reply:
`standard_inputs = (inps, dataset_template)`